pkg/server/router.go

```diff
@@ -78,6 +79,9 @@ func (r *Router) Routes(h router.Handlers) http.Handler {
 	sbDownloadHandler := supportbundle.NewDownloadHandler(r.scaled, r.options.Namespace)
 	m.Path("/v1/supportbundles/{bundleName}/download").Methods("GET").Handler(sbDownloadHandler)
 
+	kcGenerateHandler := kubeconfig.NewGenerateHandler(r.scaled)
+	m.Path("/v1/kubeconfig").Methods("POST").Handler(kcGenerateHandler)
```

Comment:
We need to access via the `/v1/harvester` aggregation APIs

Reply:
I tried the path `/v1/harvester/kubeconfig`, but got 404.